js/src/pages/price-benchmark/empty-metrics-notice/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * EmptyMetricsNotice component
+ */
+
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import EmptyMetricsGraphic from '~/images/price-benchmark/empty-metrics.svg';
+
+const EmptyMetricsNotice = () => {
+	return (
+		<div className="gla-price-benchmark__empty-metrics">
+			<img src={ EmptyMetricsGraphic } alt="" />
```

Comment:
@ankitrox Let's add the width and height of the image, i.e `88px` as per the designs.
```suggestion
			<img src={ EmptyMetricsGraphic } alt="" width={88} height={88} />
```